lib/infer/return.js

```diff
@@ -17,7 +17,14 @@ module.exports = function () {
     if (comment.returns) {
       return comment;
     }
-    var fn = finders.findTarget(comment.context.ast);
+    var fn = findTarget(comment.context.ast);
+
+    // In case of `/** */ var x = function () {}` findTarget returns
+    // the declarator.
+    if (t.isVariableDeclarator(fn)) {
```

Comment:
It still applies. The purpose of this function is to look at the ast node of the comment and infer the return tag based on the Flow return type annotation.